native-maven-plugin/src/main/java/org/graalvm/buildtools/maven/AbstractNativeImageMojo.java

```diff
@@ -243,7 +243,11 @@ protected List<String> getBuildArgs() throws MojoExecutionException {
 
         if (buildArgs != null && !buildArgs.isEmpty()) {
             for (String buildArg : buildArgs) {
-                cliArgs.addAll(Arrays.asList(buildArg.split("\\s+")));
+                if(buildArg.startsWith("\\Q") || buildArg.startsWith("-H")) {
```

Comment:
Are we sure that this is the right condition? I mean, is it sufficient that we only check start of the buildArg? Did you check all possible args starting with `\\Q` and `-H`, whether all of them can be added without `split` or not?

Reply:
the arguments that starts with \\Q and end with \\E should not be splitted by spaces.

And for also -H I see only the argument -H:ConfigurationDirectories and they should be on one line not splitted by spaces (which could be in path )